setup.py

```diff
@@ -1,48 +1,73 @@
 #!/usr/bin/env python
+# Copyright 2008 National Research Foundation (South African Radio Astronomy Observatory)
+# BSD license - see LICENSE for details
+
+from __future__ import absolute_import, division, print_function
+
+import os
+
+from setuptools import find_packages, setup
+
+
+this_directory = os.path.abspath(os.path.dirname(__file__))
+
+files = {"Readme": "README.md", "Changelog": "CHANGELOG.md"}
+
+long_description = ""
+for name, filename in files.items():
+    if not name == 'Readme':
```

Comment:
```suggestion
    if name != 'Readme':
```

Reply:
@bmerry care to explain why you prefer `!=` instead of `not ==`? 
See screenshot of perfomance below, `not x == y` performs better than `x != y`.

![image](https://user-images.githubusercontent.com/7910856/67211672-6ce8c800-f41b-11e9-933c-e77207666525.png)

Enlighten me Master :laughing:

Reply:
Try reading both statements out loud: "If not name is equal to Readme" and "If name is not equal to Readme". The second one is more readable. PEP 8 doesn't cover this case, but says the `is not` operator is preferred over `not ... is ...` for the same reason.

When I try the timeit test the results are within the noise of each other.

Reply:
"Readability counts." - Tim Peters